src/usr/local/pkg/miniupnpd.xml

```diff
@@ -192,12 +191,22 @@
 			<name>Advanced Settings</name>
 			<type>listtopic</type>
 		</field>
+		<field>
+			<fielddescr>Allow Third-Party Mapping</fielddescr>
```

Comment:
UPnP is inherently insecure but I'm not sure this functionality is worth exacerbating the issue.

Reply:
Fair enough. This should be opened as a separate feature request with an example / details of its use case.